vendor/modules.txt

```diff
@@ -0,0 +1,213 @@
+# github.com/davecgh/go-spew v1.1.0
```

Comment:
Is vendor folder in actual state? Seems no because I run `go mod vendor` and vendor folder is changed. Please recheck it

Reply:
Done. Finally Fixed.